factory/factory_pattern_real_example/README.md

```diff
@@ -0,0 +1,30 @@
+# Real world example:
+
+Here is the realworld example that I have used for one of my project with factory_pattern. 
+
+### Project Description
+
+Files
+```
+...
+- main.py --> Fast-API app
```

Comment:
we should add a direct link to these files.

Example:
- [caching.py](caching.py)